database/main.py

```diff
@@ -0,0 +1,31 @@
+# main.py
+
+import sqlalchemy as sa
+
+from fastapi import FastAPI
+from sqlalchemy.orm import sessionmaker

```

Comment:
We should organize our project in a way where each script / module has its own responsibility, this `main.py` script is supposed to handle FAST API HTTP requests' routing functions only and it should not import or deal with any SQL-related operations, that is the responsibility of our DAO modules.

You can remove sqlalchemy-related imports and all the instructions related to the session. Just trust your DAO module & function to handle all the database-related operations.